velox/type/tests/VariantTest.cpp

```diff
@@ -197,6 +197,33 @@ TEST(VariantTest, equalsWithEpsilonFloat) {
   ASSERT_FALSE(variant(sum1).equalsWithEpsilon(variant(sum3)));
 }
 
+TEST(VariantTest, mapWithNaNKey) {
```

Comment:
Can you also add a test for multimap_agg to validate ? Thanks !

Reply:
The fix for multimap_agg is in a follow up PR #9769